apps/console/src/features/applications/components/forms/general-details-form.tsx

```diff
@@ -207,7 +210,32 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
         }
     };
 
+    const {
+        data: myAccountStatus,
+        isLoading: isMyAccountStatusLoading,
+        error: myAccountStatusFetchRequestError,
+        mutate: mutateMyAccountStatusFetchRequest
+    } = getMyAccountStatus();
+
+    /**
+     * Sets the initial spinner.
+     * TODO: Remove this once the loaders are finalized.
+     */
+    useEffect(() => {
+        let status: boolean = AppConstants.DEFAULT_MY_ACCOUNT_STATUS;
+        if ( myAccountStatus ) {
+            const enableProperty = myAccountStatus["value"];
+            if ( enableProperty && enableProperty == "false" ) {
+                status = false
+            }
+        }
+        console.log(status);
+        setMyAccountStatus(status);
+    }, [ isMyAccountStatusLoading ]);
+
     return (
```

Comment:
Let's refactor for better readability.
```
if (isMyAccountStatusLoading) {
    return (
        <EmphasizedSegment padded="very">
            <ContentLoader inline="centered" active/>
        </EmphasizedSegment>
    );
}

return (
    <Form
         ...
     
```

Reply:
Addressed with 06eac07bbcb1c736313953e0cc52b62e0ac383a7